assets/activation-scripts/activate

```diff
@@ -80,7 +76,17 @@ while true; do
       FLOX_CMD="$1"
       shift
       ;;
-    --mode)
+    -e | --env)
+      shift
+      if [ -z "${1:-}" ] || [ ! -d "$1" ]; then
+        echo "Option --env requires a valid environment path as an argument." >&2
+        echo "$USAGE" >&2
+        exit 1
+      fi
+      FLOX_ENV="$1"
+      shift
+      ;;
```

Comment:
Oh ... you mean should we have the flox CLI invoke `activate --env <path>` rather than setting `$FLOX_ENV` in the environment? Yes I think that would be great, but I'd like to tackle that in another PR if that's OK? In general we have a problem with env variable leakage, and `$FLOX_ENV` is one of those that is both useful in general and problematic when it comes to nested activations. If we can avoid leaking it until where it's actually used that would be helpful.

Reply:
Yep 👍🏻